### DISCUSSION TITLE
Post here if you want to get involved!

### DISCUSSION THREAD

**Discussion by Net5F** [Net5F/AmalgamEngine#10] [Feature Development]
Post here if you're interested in getting involved. Be sure to include some information about your discipline and level of experience, and the specific feature you want to work on if you already know.

**Comment by Whystler** [comment]
I saw you mention this project on reddit, and it seemed like a few people were interested.  I came upon it by searching google for an open isometric world that might be conducive to creating an online art space.  Your idea sounded intriguing.

From what I gather, you want to make a game engine that makes it easy, through tutorials, to put up your own server that could handle a good number of players (you said 1000-3000) and run your own isometric world.  I expect that includes adding any art you like to the experience, and designing the characters (which is my interest).

I'm sorry to see you got such little support from other developers.  This would have been an interesting and fulfilling platform to use - or will be, if you and others are still working on it.

There was something at one time called Metaplace - headed up by Raph Koster.  It seemed very popular.  Not only could you use your own art, but you could get in there and code games with it.  But , as things go in virtual reality, it mysteriously and suddenly was canned despite the interest.

I hope you are able to get more support for this.  All I can offer is testing it, as I don't have any programming or networking skills.

**Reply by Net5F** [reply]
To any future people reading this, Whystler joined the discord and we started chatting there 😄
